### DISCUSSION TITLE
如何设置能开启代理端口（socks、http），但是不做透明代理，只转发到代理端口的连接？

### DISCUSSION THREAD

**Discussion by deepseer** [vernesong/OpenClash#2777] [Q&A]
类似于Windows下ClashForWindows关闭Tun Mode和System Proxy时的状态。

路由器下有多台电脑，一台上面有几个支持手动配置socks和http代理的软件（都设置了socks5://192.168.1.1:7891，还常常需要在软件内修改到其他地址或者直连），其他的则完全不需要科学工具。

**Comment by qirenzhidao** [comment]
曲线救国，局域网白名单只给走代理的设备

**Comment by chyzp** [comment]
> 类似于Windows下ClashForWindows关闭Tun Mode和System Proxy时的状态。
> 
> 路由器下有多台电脑，一台上面有几个支持手动配置socks和http代理的软件（都设置了socks5://192.168.1.1:7891，还常常需要在软件内修改到其他地址或者直连），其他的则完全不需要科学工具。

后来有办法了吗？

**Reply by deepseer** [reply]
没有。当时就换PassWall了。后来OpenClash有没有变动就没有关注过了。

**Reply by gdarmstadtiumberkelium** [reply]
https://github.com/vernesong/OpenClash/discussions/4204 这个？

**Reply by Tycho6308** [reply]
> > 类似于Windows下ClashForWindows关闭Tun Mode和System Proxy时的状态。
> > 路由器下有多台电脑，一台上面有几个支持手动配置socks和http代理的软件（都设置了socks5://192.168.1.1:7891，还常常需要在软件内修改到其他地址或者直连），其他的则完全不需要科学工具。
> 
> 后来有办法了吗？

我也在研究，ai说可以设置，但实质手搓yaml并实现不了
